### DISCUSSION TITLE
Multiline Blockquotes

### DISCUSSION THREAD

**Discussion by MARTINATINTRAVIS** [jgm/pandoc#9866] [Q&A] [answered]
Hello,

i am trying to figure out how multiline blockquote should work when using markdown as the source.
When I try the example from the manual:

```markdown
> This is a block quote. This
> paragraph has two lines.
>
> 1. This is a list inside a block quote.
> 2. Second item.
```

The output generated puts "This is a block quote. This paragraph has two lines." into one line.

I tried this offline with 3.2 and pfd as output (i tried several pdf engines).
I than tried to reproduce this on the pandoc demo page (https://pandoc.org/try/) with html as output.
This give me the following output:

```html
<blockquote>
<p>This is a block quote. This paragraph has two lines.</p>
<ol type="1">
<li>This is a list inside a block quote.</li>
<li>Second item.</li>
</ol>
</blockquote>
```
Is there something I am missing or is this a bug?

I can hack this using `> |` for blockquotes, but this is not a good solution. 
Example: 

```markdown
> | This is a block quote. This
> | paragraph has two lines.
> |
> | 1. This is a list inside a block quote.
> | 2. Second item.
```
give the following output:

```html
<blockquote>
<div class="line-block">This is a block quote. This<br />
paragraph has two lines.<br />
<br />
1. This is a list inside a block quote.<br />
2. Second item.</div>
</blockquote>
```

Does anyone have an idea how to get this working properly? 
If any vital information is missing please let me know!

Regards

**Comment by jgm** [comment, marked as answer]
By default line breaks in markdown are reflowable spaces.  You can change that globally by using `-f markdown+hard_line_breaks`.

If you only want this for the block quote, then perhaps just use explicit line breaks (`\` at end of line).

**Reply by MARTINATINTRAVIS** [reply]
Thank you for your answer. To me the example in the Docs was just really misleading.
